### PR TITLE
compat: change time style to yyyy-mm-dd

### PR DESCRIPTION
**Pull request by nishinji**
Follow ISO 8601.
`yyyy-mm-dd` is defined in ISO. The compat-repo must be changed at the same time.

### CODE REVIEW

**Review by Zangetsu38 — APPROVED** [review]

**Review by Macdu — APPROVED** [review]

**Comment by Macdu** [comment]
@nishinji Tell me when I can merge so you can update the compat repo at the same time.

**Comment by nishinji** [comment]
> @nishinji Tell me when I can merge so you can update the compat repo at the same time.

Whenever. I have already opened a PR in compat repo https://github.com/Vita3K/compatibility/pull/2688.
We just need to merge them at the same time(more precisely, merge the PR of the compat repo as soon as a new release is made ).

**Comment by Macdu** [comment]
Just one thought, won't that mess up with people using an older version (and in particular the android version)?

**Comment by Zangetsu38** [comment]
> Just one thought, won't that mess up with people using an older version (and in particular the android version)?

For user using old yes same for android, but only android build is probleme here, for other can just update 🤔

**Comment by nishinji** [comment]
Workaround added.
Builds prior to this commit will download compatibility every time they are launched, but this should not be a major issue.

**Comment by Zangetsu38** [comment]
> Workaround added. Builds prior to this commit will download compatibility every time they are launched, but this should not be a major issue.

Your workaround change nothing here 🤔
Is not inside XML or on build release
You juste create variable no used 🤔

File: vita3k/compat/src/compat.cpp [hunk 1]
```diff
@@ -173,7 +173,7 @@ bool update_app_compat_db(GuiState &gui, EmuEnvState &emuenv) {
     auto &lang = gui.lang.compat_db;
 
     // Get current date of last compat database updated at
-    const auto updated_at = net_utils::get_web_regex_result(latest_link, std::regex(R"(Updated at: (\d{2}-\d{2}-\d{4} \d{2}:\d{2}:\d{2}))"));
+    const auto updated_at = net_utils::get_web_regex_result(latest_link, std::regex(R"(Updated at: (\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}Z))"));
```
Comment (Zangetsu38) [inline vita3k/compat/src/compat.cpp:176]:
```suggestion
    const auto updated_at = net_utils::get_web_regex_result(latest_link, std::regex(R"(ISO Updated at: (\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}Z))"));
```

**Review by Zangetsu38 — APPROVED** [review]
lgtm

**Review by Macdu — APPROVED** [review]

**Pull request merged**